source/user-manual/reference/ossec-conf/wodle-s3.rst

```diff
@@ -379,18 +379,18 @@ A regular expression to determine if an event must be discarded. It requires a m
 Attributes:
 
 +-----------+------------------------------------------------------------------------------------------------------+
-| **field** | The event field where to apply the regex.                                                            |
+| **field** | The event field where to apply the regex                                                             |
 |           +------------------+-----------------------------------------------------------------------------------+
 |           | Default value    | N/A                                                                               |
 |           +------------------+-----------------------------------------------------------------------------------+
-|           | Allowed values   | A str containing the full field name path                                         |
+|           | Allowed values   | A str containing the full field name path relative to the fetched event           |
```

Comment:
Better to use an example rather than  saying "relative to the event" which doesn't seem to be clear enough